manifests/resource/vhost.pp

```diff
@@ -31,7 +46,8 @@
     owner   => 'root',
     group   => 'root',
     mode    => '0644',
-    source  => $source,
+    source  => $file_source,
```

Comment:
My understanding is that we can only provide either source or content as they are mutually exclusive.

See the following:

https://docs.puppet.com/puppet/latest/reference/types/file.html#file-attribute-source